src/smt/optimization_solver.h

```diff
@@ -65,13 +73,15 @@ enum OptResult
 class Objective
 {
  public:
-  Objective(Node n, ObjectiveType type);
+  Objective(Node n, ObjectiveType type, bool bv_is_signed_compare = true);
```

Comment:
the style of most variables in the repo is camelcase, so i suggest following the same style. correct me if i am wrong.